cms/templates/visibility_editor.html

```diff
@@ -64,35 +77,60 @@ <h3 class="modal-section-title">${_('Make visible to:')}</h3>
+                                            </label>
+                                        % else:
+                                            <label for="visibility-content-group-${partition["id"]}-${group["id"]}" class="label">${group["name"] | h}</label>
+                                        % endif
                                     </li>
                                 % endfor
+                            % endfor
 
-                                % for group_id in missing_group_ids:
-                                    <li class="field field-checkbox field-visibility-content-group was-removed">
-                                        <input type="checkbox" id="visibility-content-group-${group_id}" name="visibility-content-group" value="${group_id}" class="input input-checkbox" checked="checked" />
-                                        <label for="visibility-content-group-${group_id}" class="label">
-                                            ${_('Deleted Content Group')}
-                                            <span class="note">${_('Content group no longer exists. Please choose another or allow access to All Students and staff')}</span>
-                                        </label>
-                                    </li>
-                                % endfor
+                            ## Allow only one verification checkpoint to be selected at a time.
+                            % if verification_partitions:
+                                <li class="field field-checkbox field-visibility-verification">
+                                    <input type="checkbox"
+                                        id="verification-access-checkbox"
+                                        name="verification-access-checkbox"
+                                        class="input input-checkbox"
+                                        value=""
+                                        ${'checked="checked"' if selected_verified_partition_id is not None else ''}
+                                    />
+                                    <label for="verification-access-checkbox" class="label">
+                                        ${_('Verification Checkpoint')}:
+
+                                        ## TODO - a11y fix
+                                        <select id="verification-access-dropdown">
+                                            % for partition in verification_partitions:
+                                            <option
+                                                value="${partition["id"]}"
+                                                ${ "selected" if partition["id"] == selected_verified_partition_id else ""}
+                                            >${partition["name"]}</option>
+                                            % endfor
+                                        </select>
+                                    </label>
```

Comment:
Follow pattern as above, including the ```<div role="group" aria-labelledby"[ID of Heading]">``` to programmatically relate these controls.  The Verification checkpoint will need its own group role here, but we can use a label instead of a labelledby since there is no onscreen text to reference e.g.  ```<div role="group" aria-label="Verification checkpoint">```   Don't forget the ```aria-describedby``` pattern for the help text.